addon/README.md

```diff
@@ -12,6 +12,7 @@ The KubeFlow addon provides the following optional config options:
 - `DomainName` (text): Domain name that will be used to access the KubeFlow dashboard. Make sure to set up you DNS accordingly.
 - `OIDCProviderURL` (text): URL of external OIDC provider, e.g. Kubermatic Dex instance. Make sure that `DomainName` is specified if you are using this. If not provided, static user authentication will be used.
 - `OIDCSecret` (text): Secret string shared between the OIDC provider and KubeFlow. If not provided, [this default](https://github.com/kubeflow/manifests/blob/master/istio/oidc-authservice/base/params.env#L5) is used (insecure!).
+- `EnableIstioRBAC` (boolean): Enable Istio RBAC (Role Based Access Control) for multi-tenancy.
```

Comment:
I am thinking about do we want to keep the name as something like `EnableClusterRbacConfig`, so people can actually know that this is referring to the`ClusterRBACConfg` settings in KubeFlow/Istio. WDYT?

Reply:
Not sure.. `EnableClusterRbacConfig` may be more meaningful for people who already understand Kubeflow internals, whereas `EnableIstioRBAC` is more meaningful for newcomers, as it describes what it actually does.

Reply:
LGTM, we can keep this for now